app/javascript/components/__test__/App.js

```diff
@@ -1,12 +1,12 @@
-import React from 'react'
-import ReactDOM from 'react-dom'
-import Enzyme, { mount } from 'enzyme'
-import Adapter from 'enzyme-adapter-react-16'
-import App from '../App'
-
-Enzyme.configure({ adapter: new Adapter() })
-
-it('SearchNew renders without crashing', () => {
-  const div = document.createElement('div')
-  ReactDOM.render(<SearchNew />, div)
-})
+// import React from 'react'
```

Comment:
This file should be deleted if you don't intend to keep it. Or if you need to disable this test temporarily, use `test.skip()` 

https://jestjs.io/docs/en/api#testskipname-fn

Reply:
deleted for now, will come back for Enzyme testing later